modules/light-clients/08-wasm/types/contract_api.go

```diff
@@ -28,6 +28,7 @@ type QueryMsg struct {
 type StatusMsg struct{}
 
 // ExportMetadataMsg is a queryMsg sent to the contract to query the exported metadata of the wasm client.
+// DEPRECATED: core IBC will export all metadata.
 type ExportMetadataMsg struct{}
 
```

Comment:
I think we can just remove the `ExportMetadata` api now. It won't be used and thus doesn't need to be implemented. Left off for now as to discuss as a team